source/game/piece.py

```diff
@@ -19,9 +20,8 @@ def __init__(self, player, width, height, piece_type):
         :param piece_type: What type of piece it is.
         :type piece_type: int
         """
+        Element.__init__(self, width, height)
```

Comment:
`super().__init__()` can be used to call the super's methods. Also, this leads to better results when (if) we refactor Piece to have a different super class.